zuul-core/src/test/java/com/netflix/zuul/init2/package-info.java

```diff
@@ -0,0 +1,5 @@
+/**
+ * This package is used to test cases that we define multi filter packages in the configuration file.
+ * @author bill 2018-05-13 01:03
+ */
+package com.netflix.zuul.init2;
```

Comment:
Why is this file necessary?

Reply:
No, it's not. It's personal habbit. Should I delete it in another commit?